mcs/tools/al/Makefile

```diff
@@ -2,7 +2,7 @@ thisdir = tools/al
 SUBDIRS = 
 include ../../build/rules.make
 
-LOCAL_MCS_FLAGS = -r:Mono.Security.dll
+LOCAL_MCS_FLAGS = -r:Mono.Security.dll -r:System.Security.dll -r:Mono.CompilerServices.SymbolWriter
```

Comment:
AFAICS, this is required. I found only one relevant define - `NO_SYMBOL_WRITER`, but it isn't enough to not require referencing this assembly.